wrappers/python/examples/readme.md

```diff
@@ -11,11 +11,8 @@ These Examples demonstrate how to use the python wrapper of the SDK.
 5. [Realsense Backend](./pybackend_example_1_general.py) - Example of controlling devices using the backend interface
 6. [Read bag file](./read_bag_example.py) - Example on how to read bag file and use colorizer to show recorded depth stream in jet colormap.
 7. [Box Dimensioner Multicam](./box_dimensioner_multicam/box_dimensioner_multicam_demo.py) - Simple demonstration for calculating the length, width and height of an object using multiple cameras.
-8. [T265 Basic](./t265_example.py) - Demonstrates how to retrieve pose data from a T265
-9. [T265 Coordinates](./t265_rpy.py) - This example shows how to change coordinate systems of a T265 pose
-10. [T265 Stereo](./t265_stereo.py) - This example shows how to use T265 intrinsics and extrinsics in OpenCV to asynchronously compute depth maps from T265 fisheye images on the host.
-11. [Realsense over Ethernet](./ethernet_client_server/README.md) - This example shows how to stream depth data from RealSense depth cameras over ethernet.
-12. [D400 self-calibration demo](./depth_auto_calibration_example.py) - Provides a reference implementation for D400 Self-Calibration Routines flow. The scripts performs On-Chip Calibration, followed by Focal-Length calibration and finally, the Tare Calibration sub-routines. Follow the [White Paper Link](https://dev.intelrealsense.com/docs/self-calibration-for-depth-cameras) for in-depth description of the provided calibration methods.
+8. [Realsense over Ethernet](./ethernet_client_server/README.md) - This example shows how to stream depth data from RealSense depth cameras over ethernet.
```

Comment:
Realsense over Ethernet?

Reply:
Don't know what that is, but I'll check after

Reply:
There is an example like this!

Reply:
Yes.. needs to be removed.. next PR